localflavor/br/forms.py

```diff
@@ -137,7 +109,7 @@ class BRCNPJField(CharField):
     Otherwise both formats will be valid.
 
     .. _Brazilian CNPJ: http://en.wikipedia.org/wiki/National_identification_number#Brazil
-    .. versionchanged:: 1.4
+    .. versionchanged:: 2.1
```

Comment:
You shouldn't remove the old version changed marker because it's useful for people to know the history. I realize that the 1.4 directive is missing the explanation but it's still helpful because people can check the changelog if they need more information about the change.